receiver/awsxrayreceiver/README.md

```diff
@@ -0,0 +1,68 @@
+# AWS X-Ray Receiver
+
+**Status: alpha**
```

Comment:
Please add a note saying that this component cannot be used since it is not yet fully implemented.